test/environments/wrappers/wrappers.jl

```diff
@@ -37,6 +37,9 @@
             n -= 1
             @test n >= 0
         end
+
+        RLBase.reset!(env′)
+        @test env′.current_t == 1
```

Comment:
```suggestion
        @test is_terminated(env′) == false
```

Reply:
We want both, right?
`@test env′.current_t == 1` and `@test is_terminated(env′) == false` after `reset!`

Reply:
Yeah, but usually we only test the exposed interfaces. So that the test cases are more robust in case the inner field name is renamed. This is just a suggestion. Feel free to merge it.

Reply:
Okay. I see. I'll keep them both for now because I have tripped on the `current_t` test a couple of times earlier.